crates/masonry/README.md

```diff
@@ -1,157 +1,41 @@
-(On ubuntu-based distro, running `sudo apt-get install libgtk-3-dev` from the terminal will do the job.)
-
-### OpenBSD
-
-On OpenBSD, Masonry requires gtk+3; install from packages:
-```sh
-pkg_add gtk+3
-```
-
-## Example
-
-The todo-list example looks like this:
-
-```rust
-use masonry::widget::{prelude::*, TextBox};
-use masonry::widget::{Button, Flex, Label, Portal, WidgetMut};
-use masonry::Action;
-use masonry::{AppDelegate, AppLauncher, DelegateCtx, WindowDescription, WindowId};
-
-const VERTICAL_WIDGET_SPACING: f64 = 20.0;
-
-struct Delegate {
-    next_task: String,
-}
-
-impl AppDelegate for Delegate {
-    fn on_action(
-        &mut self,
-        ctx: &mut DelegateCtx,
-        _window_id: WindowId,
-        _widget_id: WidgetId,
-        action: Action,
-    ) {
-        match action {
-            Action::ButtonPressed => {
-                let mut root: WidgetMut<Portal<Flex>> = ctx.get_root();
-                let mut flex = root.child_mut();
-                flex.add_child(Label::new(self.next_task.clone()));
-            }
-            Action::TextChanged(new_text) => {
-                self.next_task = new_text.clone();
-            }
-            _ => {}
-        }
-    }
-}
+[![Latest published version.](https://img.shields.io/crates/v/masonry.svg)](https://crates.io/crates/masonry)
+[![Documentation build status.](https://img.shields.io/docsrs/masonry.svg)](https://docs.rs/masonry)
+[![Apache 2.0 license.](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](#license)
 
-fn main() {
-    // The main button with some space below, all inside a scrollable area.
-    let root_widget = Portal::new(
-        Flex::column()
-            .with_child(
-                Flex::row()
-                    .with_child(TextBox::new(""))
-                    .with_child(Button::new("Add task")),
-            )
-            .with_spacer(VERTICAL_WIDGET_SPACING),
-    );
+[![Linebender Zulip chat.](https://img.shields.io/badge/Linebender-%23masonry-blue?logo=Zulip)](https://xi.zulipchat.com/#narrow/stream/317477-masonry)
+[![GitHub Actions CI status.](https://img.shields.io/github/actions/workflow/status/linebender/xilem/ci.yml?logo=github&label=CI)](https://github.com/linebender/xilem/actions)
+[![Dependency staleness status.](https://deps.rs/crate/masonry/latest/status.svg)](https://deps.rs/crate/masonry)
 
-    let main_window = WindowDescription::new(root_widget)
-        .title("To-do list")
-        .window_size((400.0, 400.0));
+</div>
 
-    AppLauncher::with_window(main_window)
-        .with_delegate(Delegate {
-            next_task: String::new(),
-        })
-        .log_to_console()
-        .launch()
-        .expect("Failed to launch application");
-}
-```
+Masonry gives you a platform to create windows (using [winit] as a backend) each with a tree of widgets.
+It also gives you tools to inspect that widget tree at runtime, write unit tests on it, and generally have an easier time debugging and maintaining your app.
 
```

Comment:
Do we also want to mention a similar thing of alpha status to Vello (maybe not as a GitHub alert, though)

Stuff like:
- text input being somewhat janky
- snapshot testing not being consistent cross platform

Reply:
Added a note about it being alpha.